strategy/json/sql/postgres/manager.go

```diff
@@ -144,15 +144,18 @@ func (m *SingleStreamManager) NewAggregateProjector(
 		return nil, err
 	}
 
+	var queue driverSQL.NotificationQueuer
+	//TODO:: initialise Queue or SmartQueue based on value of retryDelay
+	queue = driverSQL.NewNotificationQueue(30, *retryDelay, m.metrics)
```

Comment:
Implementation for `Queue` and `SmartQueue` are in a separate PR https://github.com/hellofresh/goengine/pull/54